src/components/SecondaryPanes/Frames/FrameMenu.js

```diff
@@ -6,37 +6,53 @@ import type { ContextMenuItem } from "debugger-html";
 import { kebabCase } from "lodash";
 
 function formatMenuElement(
-  labelString: string,
-  accesskeyString: string,
+  label: string,
+  accesskey: string,
   click: Function,
   disabled: boolean = false
 ): ContextMenuItem {
-  const label = L10N.getStr(labelString);
   const id = `node-menu-${kebabCase(label)}`;
   return {
     id,
     label,
-    accesskey: L10N.getStr(accesskeyString),
+    accesskey,
     disabled,
     click
   };
 }
 
 function copySourceElement(url) {
-  return formatMenuElement("copySourceUrl", "copySourceUrl.accesskey", () =>
-    copyToTheClipboard(url)
+  return formatMenuElement(
+    L10N.getStr("copySourceUrl"),
+    L10N.getStr("copySourceUrl.accesskey"),
+    () => copyToTheClipboard(url)
   );
 }
 
 function copyStackTraceElement(copyStackTrace) {
-  return formatMenuElement("copyStackTrace", "copyStackTrace.accesskey", () =>
-    copyStackTrace()
+  return formatMenuElement(
+    L10N.getStr("copyStackTrace"),
+    L10N.getStr("copyStackTrace.accesskey"),
+    () => copyStackTrace()
+  );
+}
+
+function toggleFrameworkGroupingElement(
+  toggleFrameworkGrouping,
+  frameworkGroupingOn
+) {
+  const actionType = frameworkGroupingOn ? "Disable" : "Enable";
+  return formatMenuElement(
+    L10N.getFormatStr("framework.toggleGrouping", actionType),
```

Comment:
we have support for `getFormatStr` but I try to avoid it as it makes it more complicated. i.e. "Disable" wouldn't be translated here. more over languages are hard:

```js
const key = frameworkGroupingOn ? "framework.toggleGroupingOn" : "framework.toggleGroupingOff"
formatMenuElement(key, toggleFrameworkGrouping)
```